### PR TITLE
Update neofs-contract to v0.20.0

### PR DESCRIPTION
**Pull request by roman-khimov**
As a part of https://github.com/nspcc-dev/neofs-contract/issues/384 I'd like to unify different code dealing with them.

### CODE REVIEW

**Comment by roman-khimov** [comment]
@evgeniiz321, tests fail here and I can't get any logs.

**Comment by roman-khimov** [comment]
This updates contracts as well, so it's gonna be postponed till we have a proper 0.20.0.

**Review by carpawell — CHANGES_REQUESTED** [review]
Tests adoption.

File: cmd/neofs-adm/internal/modules/morph/initialize_deploy.go [hunk 1]
```diff
@@ -429,12 +435,14 @@ func readContractsFromArchive(file io.Reader, names []string) (map[string]*contr
 		}
 
 		switch {
-		case strings.HasSuffix(h.Name, filepath.Join(ctrName, ctrName+"_contract.nef")):
+		case strings.HasSuffix(h.Name, filepath.Join(ctrName, ctrName+"_contract.nef")) ||
+			strings.HasSuffix(h.Name, "contract.nef"):
```
Comment (cthulhu-rider) [inline cmd/neofs-adm/internal/modules/morph/initialize_deploy.go:439]: 2nd operand is enough

File: cmd/neofs-adm/internal/modules/morph/initialize_deploy.go [hunk 2]
```diff
@@ -372,11 +372,17 @@ func (c *initializeContext) readContracts(names []string) error {
 func readContract(ctrPath, ctrName string) (*contractState, error) {
 	rawNef, err := os.ReadFile(filepath.Join(ctrPath, ctrName+"_contract.nef"))
 	if err != nil {
-		return nil, fmt.Errorf("can't read NEF file for %s contract: %w", ctrName, err)
+		rawNef, err = os.ReadFile(filepath.Join(ctrPath, "contract.nef"))
```
Comment (carpawell) [inline cmd/neofs-adm/internal/modules/morph/initialize_deploy.go:375]: go this way only if file not found? also, do you play to drop it in the future?

File: pkg/morph/client/netmap/netmap.go [hunk 3]
```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 
 	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
-	netmapcontract "github.com/nspcc-dev/neofs-contract/contracts/netmap"
+	netmaprpc "github.com/nspcc-dev/neofs-contract/rpc/netmap"
```
Comment (cthulhu-rider) [inline pkg/morph/client/netmap/netmap.go:7]: i liked previous alias more tbh

**Review by cthulhu-rider — COMMENTED** [review]
good overall

File: cmd/neofs-adm/internal/modules/morph/initialize_deploy.go (hunk 2 above)
Comment (cthulhu-rider) [reply cmd/neofs-adm/internal/modules/morph/initialize_deploy.go:375]: @carpawell is faster, but i'll leave my comment anyway

err may be other than 404, so i'd either:

1. try other file on 404 only
2. try anyway, but keep both errors when it fails too

same for manifest files

File: cmd/neofs-adm/internal/modules/morph/initialize_deploy.go (hunk 1 above)
Comment (roman-khimov) [reply cmd/neofs-adm/internal/modules/morph/initialize_deploy.go:439]: Yep, fixed.

File: cmd/neofs-adm/internal/modules/morph/initialize_deploy.go (hunk 2 above)
Comment (carpawell) [reply cmd/neofs-adm/internal/modules/morph/initialize_deploy.go:379]: > also, do you plan to drop it in the future?

why do we try to support both ways? how long?

Comment (roman-khimov) [reply cmd/neofs-adm/internal/modules/morph/initialize_deploy.go:379]: Because it still is a generic tool. We'll see. An issue can be created.

**Review by carpawell — APPROVED** [review]

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**